src/daft-csv/src/read.rs

```diff
@@ -136,6 +138,49 @@ pub fn read_csv_bulk(
     tables.into_iter().collect::<DaftResult<Vec<_>>>()
 }
 
+#[inline]
+fn assert_stream_send<'u, R>(
+    s: impl 'u + Send + Stream<Item = R>,
+) -> impl 'u + Send + Stream<Item = R> {
+    s
+}
+
+// Parallel version of table concat
+// get rid of this once Table APIs are parallel
+fn parallel_table_concat(tables: &[Table]) -> DaftResult<Table> {
```

Comment:
Nit: The table concatenation is an implementation detail that probably doesn't need to go into the function name, any particular reason you wanted to explicitly call its parallel nature out?
```suggestion
fn concat_tables(tables: &[Table]) -> DaftResult<Table> {
```

Reply:
We already have a `Table::concat` so wanted to call it out, but it probably makes sense to drop it from the name and add it as a comment